anaconda.spec.in

```diff
@@ -467,6 +467,10 @@ rm -rf \
 %{_prefix}/libexec/anaconda/dd_*
 
 %changelog
+* Fri Apr 19 2024 Katerina Koukiou <kkoukiou@redhat.com> - 40.22.3.1-1
+- Get latests changes from fedora-40 branch to RHEL-10, Resolves: RHEL#33385)
```

Comment:
Is there a reason for "#" in the RHEL issue ID? Could you change it to "RHEL-33385" instead?